tests/unit/test_event_timer.py

```diff
@@ -0,0 +1,44 @@
+#  Copyright 2024 Canonical Ltd.
+#  See LICENSE file for licensing details.
+import logging
+import secrets
+
+from event_timer import EventTimer
+
+
+def test_is_active_true():
+    """
+    arrange: create an EventTimer object and mock exec command to return zero exit code
+    act: call is_active
+    assert: return True
+    """
+    evt = EventTimer(secrets.token_hex(16))
+    assert evt.is_active(secrets.token_hex(16))
```

Comment:
Maybe `evt.is_active(event_name=secrets.token_hex(16))`  makes it clearer that you are using a generated string as stub event name.
Same applies to other usage of `secrets.token_hex` in this file.